server/middleware/prepaids.coffee

```diff
@@ -116,3 +124,90 @@ module.exports =
           schoolPrepaidsMap[school].push prepaid
 
     res.send({prepaidActivityMap, schoolPrepaidsMap})
+  
+  # Separate endpoint from legacy prepaid purchase handler
+  purchaseStarterLicenses: wrap (req, res) ->
+    if req.body.type not in ['starter_license']
+      throw new errors.Forbidden("License type invalid: #{req.body.type}")
+
+    creator = req.user
+    maxRedeemers = parseInt(req.body.maxRedeemers)
+    months = parseInt(req.body.months)
+    token = req.body.stripe?.token
+    timestamp = req.body.stripe?.timestamp
+
+    if isNaN(maxRedeemers) or maxRedeemers < 1
+      throw new errors.UnprocessableEntity("Invalid number of licenses to buy: #{maxRedeemers}")
+
+    alreadyOwnedStarterLicenses = yield Prepaid.find({
+      creator: mongoose.Types.ObjectId(creator.id)
```

Comment:
Ah, that explains some things. I was always confused how some things used one and some used the other, mostly interchangeably.